CHANGELOG.md

```diff
@@ -1,5 +1,53 @@
 # filecoin-ffi changelog
 
+## 0.27.0
+
+This release migrates from specs-actors 4.0.1 to 0.5.4.
```

Comment:
`4.0.1` doesn't look like the right prior version tag.

Reply:
@anorth 

> 4.0.1 doesn't look like the right prior version tag.

Thanks. It should read `0.4.1`